Explorer/Assets/DCL/Browser/DecentralandUrls/DecentralandUrlsSource.cs

```diff
@@ -40,6 +40,7 @@ private static string RawUrl(DecentralandUrl decentralandUrl) =>
                 DecentralandUrl.ApiAuth => $"https://auth-api.decentraland.{ENV}",
                 DecentralandUrl.AuthSignature => $"https://decentraland.{ENV}/auth/requests",
                 DecentralandUrl.POI => $"https://dcl-lists.decentraland.{ENV}/pois",
+                DecentralandUrl.PlacesByCategory => $"https://places.decentraland.{ENV}/api/map",
```

Comment:
We have repeated the places api url. We could expose the `DecentralandUrl.ApiPlaces` as:
```
https://places.decentraland.{ENV}/api
```
Then append the path on specific queries. Otherwise we will keep adding entries here